pkgs/servers/emby/default.nix

```diff
@@ -36,7 +36,7 @@ stdenv.mkDerivation rec {
 
```

Comment:
Does actually really all files of emby should be in `$out/bin`? It seems to me that only `$out/bin/MediaBrowser.Server.Mono` belongs here